OsuPlayer.Tests/OsuPlayer.Network/GitHubUpdaterTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using NUnit.Framework;
+using OsuPlayer.Network;
+
+namespace OsuPlayer.Tests.OsuPlayer.Network;
+
+public class GitHubUpdaterTests
+{
+    [Test]
+    public async Task CheckForUpdatesTest()
```

Comment:
This apparently runs into problems where the github api limits trigger and we can't get a valid response. So likely removing this test completely for now would be best.

Reply:
Yeah makes sense saw that error already. Lets remove them for now